### PR TITLE
Update UPGRADE.md

### PR DESCRIPTION
**Pull request by nadeeraka**
typo fixed

### CODE REVIEW

**Comment by fzaninotto** [comment]
Thanks! It seems you somehow changed the indentation, too. Can you revert that change?

**Comment by nadeeraka** [comment]
i will

**Comment by nadeeraka** [comment]
done

File: UPGRADE.md [hunk 1]
```diff
@@ -1240,7 +1240,7 @@ const App = () => (
         })}
     >
         <TranslationProvider>
-+           <DataProviderContext.Provider value={dataProvider} />
++           <DataProviderContext.Provider valuse={dataProvider} >
```
Comment (djhi) [inline UPGRADE.md]:
```suggestion
+           <DataProviderContext.Provider value={dataProvider}>
```

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**